registry.json

```diff
@@ -3812,5 +3812,19 @@
     "telegram": "monadapool",
     "operatorName": "Anton",
     "github": ""
+  },
+  {
+    "ticker": "ALND",
+    "poolId": "6f223af7c6069f7428cf7605efba84ec410f5476550f83dbb3096238",
+    "social": {
+      "website": "https://www.allnodes.com/ada/staking",
```

Comment:
I am unable to tell if this is one stakepool or a non-custodial stakepool as a service platform.... Please can you confirm?

Reply:
Sorry. What do you mean? We have one Cardano stake pool. Not multiple.